app/res/values-v11/themes.xml

```diff
@@ -81,6 +81,8 @@
         <item name="android:paddingLeft">@dimen/title_paddingLeft</item>
 	</style>
     
-    
+    <style name="Theme.Dialog" parent="@android:style/Theme.Holo.Light.Dialog">

```

Comment:
@lu16j Thanks for the reply! I think we figured it out-- the layout for the PrintActivity is very simple, just a small progress spinner and nothing else, so it looks like you were just trying to remove the title bar from that activity, since it would look odd.